cpp/harvest/marc_issn_lookup.cc

```diff
@@ -148,6 +148,13 @@ struct CacheEntry {
                 else
                     preferred_title_ = "";
             }
+
+            if (field.getTag() == "776") {
+                if (not field.getFirstSubfieldWithCode('x').empty()) {
+                    preferred_issn_ = StringUtil::ASCIIToUpper(field.getFirstSubfieldWithCode('x'));
+                    InsertIssnIfNotExist(preferred_issn_, &issns_);
```

Comment:
Insert is OK, but please use the value from x directly (uppercase) instead of the class member.